src/sql/azdata.proposed.d.ts

```diff
@@ -1934,4 +1934,21 @@ declare module 'azdata' {
 			isPrimary: boolean;
 		}
 	}
+
+	export interface ModelBuilder {
+		chart(): ComponentBuilder<ChartComponent, ChartComponentProperties>;
+	}
+
+	export interface ChartComponentProperties extends ComponentProperties {
+		chartType?: string | undefined;
+		data?: number[] | undefined;
+		labels?: string[] | undefined;
```

Comment:
These attributes can go in doughchartdata interface

Reply:
moved these attributes to specific chart properties